src/SFA.DAS.Commitments.Application.UnitTests/Commands/CreateCommitment/WhenCreatingCommitment.cs

```diff
@@ -57,7 +57,7 @@ public void SetUp()
                 .With(x => x.FirstName, "First name")
                 .With(x => x.FirstName, "Last name")
                 .With(x => x.ProviderRef, "Provider ref")
-                .With(x => x.EmployerRef, null)
+                .With(x => x.EmployerRef, (string) null)
```

Comment:
When building the new code i found a compiler error with this line where it could not tell if the second parameter was a string of a Func<string> so I've declared it as a string to resolve the ambiguity.